modules/nodes-containers-volumes-removing.adoc

```diff
@@ -51,17 +51,235 @@ For example:
 ----
 $ oc set volume dc/d1 --remove --name=v1
 ----
++
```

Comment:
My $0.02 is to skip anything that performs a removal.  A customer who is interested in GitOpts and declarative configs would just delete the the volume from the original YAML and reapply it.